DAO.sol

```diff
@@ -821,7 +821,8 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
```

Comment:
Add a comment here and in the interface for `halveMinQuorum()` stating that the curator/s can now halve minQuorum in case of an emergency.